docs/src/main/asciidoc/security-basic-authentication-howto.adoc

```diff
@@ -15,23 +15,37 @@ Enable xref:security-basic-authentication.adoc[Basic authentication] for your Qu
 
 == Prerequisites
 
-* You have installed at least one extension that provides an `IdentityProvider` based on username and password, such as xref:security-jdbc.adoc[Elytron JDBC].
+* You have installed at least one extension that provides an `IdentityProvider` based on username and password.
+For example:
+
+* xref:security-properties[Elytron security properties file extension `(quarkus-elytron-security-properties-file)`]
```

Comment:
I'd like to suggest to link to the security-jpa instead as this one is used for tests, where everything can be configured in the properties

Reply:
+1 to add security-jpa, but I would keep quarkus-elytron-security-properties-file in the list for completness, there can be hint saying that this is suggested for testing purposes, not for production

Reply:
+1 to @rsvoboda's suggestion.

Reply:
@jsmrcka , @sberyozkin , @rsvoboda - Thanks so much for reviewing and suggestions.

Regarding the Jakarta Persistence extension. For alignment with the other guidance we give the reader,  I also added the REST extension also as we recommend this too:

How does this look:

![image](https://github.com/quarkusio/quarkus/assets/92924207/5a759c83-382f-4268-8b42-38766d6b2e9f)

Or is only one JPA extension needed here? WDYT?

Reply:
Let's keep it as is, looks good to me.